traffic_ops/app/lib/API/Server.pm

```diff
@@ -400,40 +401,25 @@ sub get_servers_by_status {
 	my $orderby           = $self->param('orderby') || "hostName";
 	my $orderby_snakecase = lcfirst( decamelize($orderby) );
 
+	my $forbidden;
```

Comment:
The function was previously testing the ds_user table as well and filtering by it.
We agreed it should be removed and replaced  only with "is-oper" test